pokedex/storage.lua

```diff
@@ -20,8 +23,13 @@ local storage_settings = {}
 local initialized = false
 
 
-LOCATION_PC = 0
-LOCATION_PARTY = 1
+local LOCATION_PC = 0
+local LOCATION_PARTY = 1
+
+local LATEST_VERSION = 2
+
+-- Default storage data
+local _storage_data = {player_pokemon={}, counters={}, sorting={}, settings={max_party_pokemon=3}, version=LATEST_VERSION}
```

Comment:
I would call this `DEFAULT_STORAGE_DATA` and lose the comment. It's never supposed to be changed, just deep copied, so I would make that clear by using all caps.

Reply:
Is it okay for `counters` to be an empty table here? The previous code seemed to go through pains to ensure `counters` was populated.